plugins/doc_fragments/proxmox.py

```diff
@@ -35,13 +35,23 @@ class ModuleDocFragment(object):
       - Specify the token secret.
     type: str
     version_added: 1.3.0
+  api_backend:
+    description:
+      - Specify the API backend used for connecting.
+    type: str
+    choices: [ 'https', 'openssh' ]
+    default: https
+    version_added: 4.5.0
   validate_certs:
     description:
       - If C(no), SSL certificates will not be validated.
       - This should only be used on personally controlled sites using self-signed certificates.
     type: bool
     default: no
-requirements: [ "proxmoxer", "requests" ]
+requirements:
+  - `proxmoxer`
+  - `requests`
+  - `openssh_wrapper` (optional, only required when using `api_backend: openssh`)
```

Comment:
```suggestion
  - openssh_wrapper (optional, only required when using I(api_backend=openssh))
```
Please remove the backticks from the two lines above.